.circleci/config.yml

```diff
@@ -611,3 +612,29 @@ jobs:
       - run:
           name: "Run centos-cpp image"
           command: docker-compose run -e NUM_THREADS=16 --rm centos-cpp
+
+  docker-compose-presto-native:
+    machine: # Use machine executor type to use docker-compose
+      image: ubuntu-2204:2022.10.1
+    resource_class: 2xlarge
+    steps:
+      - run:
+          name: "Checkout prestodb"
+          command: git clone git@github.com:prestodb/presto.git
```

Comment:
```suggestion
          command: git clone https://github.com/prestodb/presto.git
```

Reply:
That part succeeds though, the failure is inside the container job. Let me see if PRESTODB_REPOSITORY=https/... works.